src/Microsoft.ML.Data/DataLoadSave/Text/LoadColumnAttribute.cs

```diff
@@ -8,14 +8,14 @@
 namespace Microsoft.ML.Data
 {
     /// <summary>
-    /// Describes column information such as name and the source columns indices that this
-    /// column encapsulates.
+    /// Allow member to specify mapping to column(s) in text file.
```

Comment:
>column [](start = 43, length = 6)

Please do not use the word "column" here since that has an overloaded context in `IDataView` itself.

What I tend to favor, given that a different term is required, is what appears in [Wikipedia's description of CSV](https://en.wikipedia.org/wiki/Comma-separated_values), specifically, "field."

Reply:
Here and everywhere. I notice this is a problem with all the new documentation is this bad usage of the word "column." This would be troublesome anywhere, but it is especially troublesome since this is literally on a attribute where column is used to describe a rather entirely different concept. :P

---
In reply to: [258283784](https://github.com/dotnet/machinelearning/pull/2638#discussion_r258283784) [](ancestors = 258283784)